cypress/integration/sales/credit_memo_price_difference_for_sales_invoice.js

```diff
@@ -0,0 +1,229 @@
+/*
+ * #%L
+ * metasfresh-pharma
```

Comment:
```suggestion
 * metasfresh-e2e
```

good idea to add the licence header, but pls add `metasfresh-e2e`, and the year 2019 (instead of 2018)